code/game/gamemodes/changeling/changeling.dm

```diff
@@ -289,7 +289,8 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 	var/changelingID = "Changeling"
 	var/geneticdamage = 0
 	var/isabsorbing = 0
-	var/geneticpoints = 10
+	var/geneticpoints = 5 //Currency used for purchasing powers - gained from absorption
+	var/total_genetic_points = 5 //Used in respeccing to reset points to the proper amount

```

Comment:
tbh it should track how much they've spent, and then reset them back to that amount of points.
absorbing people so you have 10 points then respeccing down to 5 is just pain.